CefSharp.Core/Internals/ClientAdapter.cpp

```diff
@@ -290,6 +307,28 @@ namespace CefSharp
             }			
         }
 
+        void ClientAdapter::OnResourceRedirect(CefRefPtr<CefBrowser> browser, CefRefPtr<CefFrame> frame, const CefString& oldUrl, CefString& newUrl)
+        {
+            IRequestHandler^ handler = _browserControl->RequestHandler;
+            if (handler != nullptr)
+            {
+                auto managedNewUrl = StringUtils::ToClr(newUrl);
+                handler->OnResourceRedirect(_browserControl, frame->IsMain(), StringUtils::ToClr(oldUrl), managedNewUrl);
+
+                //TODO: Determine if there is a better way to pass the reference in rather than reassigning the string
+                newUrl = StringUtils::ToNative(managedNewUrl);
```

Comment:
The `ref` part works without any extra symbols. I was hoping to avoid converting to a `managed string`, letting the user assign the string that's passed by `ref` and then converting back to `CefString`.